app/views/todos/show.html.haml

```diff
@@ -1,7 +0,0 @@
-.container
```

Comment:
If you can put the file back in with simple styling. People can't hit the link, but people familar with rails will try and hit the direct route if they are curious.